src/orders/components/OrderTransaction/OrderTransaction.module.css

```diff
@@ -0,0 +1,13 @@
+/* Chevron rotation for accordion */
+.chevron {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: transform 0.2s ease;
+  transform: rotate(-90deg);
+}
+
+button[data-state="open"] .chevron,
+[data-state="open"] .chevron {
+  transform: rotate(0deg);
+}
```

Comment:
issue: I think we should instead create a new component in macaw or if it's very specific to Dashboard, a component like `<DashbordAccordeon>` with this style override.

Adding too many overrides can cause style differences really fast.

Perhaps we should replace old MaterialUI components, this way we won't need overrides

Reply:
I don't want to block anything with now refactoring things to Macaw tbh. But it may be a good follow up issue.